### ISSUE TITLE
Is the output of fine-tuning the outdoor model using a depth map also a depth map?

### ISSUE THREAD

**Issue by HaosenZ** [DepthAnything/Depth-Anything-V2#96]
Thank you for your contribution
I saw in your metric depth description that the output of the pre trained model can be used as a disparity map. Now, I want to use a custom dataset that includes RGB images and a 16 bit depth map to fine tune the outdoor model. Is the resulting output a depth map?

**Comment by Eyvaz27** [comment]
If you finetune the model against gt depth maps while following https://github.com/DepthAnything/Depth-Anything-V2/issues/102#issuecomment-2275132770 results should be depth predictions (furthermore metric_depth/train.py is designed to fine-tune architecture wrt gt depth maps)
